experiments/zmx-supervisor/internal/supervisor/child.go

```
@@ -0,0 +1,93 @@
+	marker := ExitMarker{Version: 1, SessionID: sessionID, StartedAt: now}
+	if len(command) == 0 {
+		marker.Error = "empty child command"
+		exited := time.Now().UTC()
+		marker.ExitedAt = &exited
+		code := 127
+		marker.ExitCode = &code
+		_ = WriteExitMarker(markerPath, marker)
+		return code
+	}
+
+	child := exec.Command(command[0], command[1:]...)
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)